packages/frint-router/src/makeRouterService.spec.js

```diff
@@ -113,13 +113,21 @@ describe('frint-router › makeRouterService', function () {
 
             // second push
             null,
+
+            // third push
+            {
+              url: '/about',
+              isExact: true,
+              params: {},
+            },
```

Comment:
I'm missing here testing that caching actually works. I'd suggest that it should test if this match is actually the same instance (depending on enableCache option, so at least two tests).